package.json

```diff
@@ -5,15 +5,15 @@
   "main": "index.js",
   "dependencies": {
     "csslint": "^0.10.0",
-    "event-stream": "^3.1.2",
-    "gulp-util": "^3.0.0"
+    "event-stream": "^3.3.0",
+    "gulp-util": "^3.0.4"
   },
   "devDependencies": {
-    "mocha": "^1.18.2",
-    "should": "^3.3.1"
+    "mocha": "^2.2.1",
+    "should": "^5.2.0"
   },
   "scripts": {
-    "test": "./node_modules/.bin/mocha"
+    "test": "mocha"
```

Comment:
Is this going to work if `mocha` is not installed globally and paths are not setup correctly? I'd rather point at the locally installed version to be sure we're running what we installed instead of being at the mercy of paths. Feel free to set me straight if this understanding is not correct :)